src/client/styles/components/_form-dropdown.scss

```diff
@@ -16,4 +16,14 @@
   &.has-error {
     border-color: $red;
   }
+
+  @include media(desktop){
+    padding: 9px 0px;
```

Comment:
Don't need to add the measurement if the value is 0. Can just be `padding: 9px 0;`.